packages/zowe-explorer/src/command/TsoCommandHandler.ts

```diff
@@ -247,7 +249,17 @@ export class TsoCommandHandler extends ZoweCommandProvider {
                         title: localize("issueTsoCommand.command.submitted", "TSO command submitted."),
                     },
                     () => {
-                        return ZoweExplorerApiRegister.getCommandApi(profile).issueTsoCommand(command, acctNum);
+                        if (ZoweExplorerApiRegister.getCommandApi(profile).issueTsoCommandWithParms) {
+                            return ZoweExplorerApiRegister.getCommandApi(profile).issueTsoCommandWithParms(
+                                command,
+                                tsoParams
+                            );
+                        } else {
```

Comment:
I was able to get it working for the extender with the old API by adding this check on tsoParams here before the call to the old API
```
let acctNum: string;
if (!tsoParams) {
    acctNum = undefined;
} else {
    acctNum = tsoParams.account;
}
```

Reply:
followed by 
```
return ZoweExplorerApiRegister.getCommandApi(profile).issueTsoCommand(
                                command,
                                acctNum
                            );
```